src/Dfc.CourseDirectory.WebV2/Validation/ValidatorBaseExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using GovUk.Frontend.AspNetCore;
+
+namespace Dfc.CourseDirectory.WebV2.Validation
+{
+    public static class ValidatorBaseExtensions
+    {
+        public static IRuleBuilderInitial<T, Date?> Date<T>(
```

Comment:
It would be nice to add xmldoc here with an example usage that would then show up in intellisense